Make error plugin configurable (#419)
php-http/HttplugBundle: CHANGELOG.md

The change log describes what is "Added", "Removed", "Changed" or "Fixed" between each release.

> # 1.27.0 -

Reviewer: ```suggestion
# 1.27.0 - 2022-07-25
```

> - Added support for configuring options of error plugin via configuration

Reviewer: ```suggestion
- Added support for configuring the error plugin via configuration
```

(before you could not even enable the plugin with default options...)
